docs/components/modeler/web-modeler/collaboration.md

```diff
@@ -28,25 +30,49 @@ There are four roles with different levels of access rights that can be assigned
 - **Commenter**: The user cannot edit folders or diagrams or invite users, but can view diagrams and properties and leave comments.
 - **Viewer**: The user cannot edit folders or diagrams nor leave comments, but can only view diagrams.
 
-Additionally, the **Owner** and **Admins** of the organization have special privileges to do administrative tasks in **super-user mode**.
+Additionally, users with elevated access have special privileges to do administrative tasks in **super-user mode**.
 
 #### Super-user mode
 
-:::note
-Super-user mode is not yet available in Web Modeler Self-Managed.
-:::
-
-Super-user mode is only available to the **Owner** and **Admins** of the organization and can be enabled via the user menu in Web Modeler:
+Super-user mode is only available to users with elevated access and can be enabled via the user menu in Web Modeler:
 
 <p><img src={SuperUserModeImg} style={{width: 280}} alt="Enable super-user mode in Web Modeler's user menu" /></p>
 
 The main purpose of this mode is to assign collaborators to orphaned projects (which have no collaborators).
 Ordinarily, these projects would not be accessible or visible to any users.
 
-When the **Owner** or an **Admin** activates super-user mode, they are temporarily granted **Project Admin** access to all projects
+When a user activates super-user mode, they are temporarily granted **Project Admin** access to all projects
 of the organization. This allows them to assign collaborators to orphaned projects and gives them
 full access when none of the ordinary collaborators are available.
 
+##### Required Roles/Permissions for Super-User Mode Access
+
+<Tabs groupId="permissions" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+
+<TabItem value='saas'>
+
+The user must be assigned the organization **Owner** or **Admin** role.
+
+</TabItem>
+
+<TabItem value='self-managed'>
+
+The user must be assigned the **Web Modeler Admin** role.
+
+If the role is not pre-existing, it can be created with the following permissions:
+
+- Web Modeler Internal API - `write:*`
+- Web Modeler Internal API - `admin:*`
+- Camunda Identity Resource Server - `read:users`
```

Comment:
💭 It could be useful to also add that the `admin:*` permission must be created by hand for the `Web Modeler Internal API` if it does not exist

Reply:
💭 I think it would also make sense to cross-link https://github.com/camunda/camunda-docs/blob/main/docs/self-managed/identity/user-guide/roles/add-assign-permission.md here

Reply:
I also added the internal permissions to the preset permission table in the identity doc